qml/FileListPage.qml

```diff
@@ -59,6 +60,18 @@ Page {
             searchField.text = ""
         }
     }
+    
+    function getSortParameterName(parameter) {
+        if (parameter === FilterModel.Name) {
+            return qsTrId("jolla-email-me-sort_name")
+        } else if (parameter === FilterModel.Type) {
+            return qsTrId("jolla-email-me-sort_type")
+        } else if (parameter === FilterModel.Date) {
+            return qsTrId("jolla-email-me-sort_date")
```

Comment:
Actually could have separate translations for these as the result is used inside a string. Might want to capitalize menu item but not these.

Reply:
Explain please

Reply:
Done